ceph_iscsi_config/client.py

```diff
@@ -664,3 +664,29 @@ def _encrypt(self):
     chap_str = property(_get_chap_str,
                         _set_chap_str,
                         doc="get/set the chap string (user/password)")
+
+
+def define_clients(logger, config):
```

Comment:
I think making this a static method of GWClient matches the style we are using where if the class needs to access the config object to return/access multiple objects of a type. If you see something else though let me know.